src/SFProvider.cc

```diff
@@ -1,12 +1,14 @@
 #include "TauAnalysisTools/TauTriggerSFs/interface/SFProvider.h"
-
+#include <boost/algorithm/string.hpp>
```

Comment:
If we can we should avoid external dependencies.

Reply:
I cleaned it all